pom.xml

```diff
@@ -128,6 +131,12 @@
 			<scope>runtime</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.package-url</groupId>
+			<artifactId>packageurl-java</artifactId>
+			<version>${packageurl.version}</version>
+		</dependency>
+

```

Comment:
This is MIT, so it's fine.
@aalmiray @rjalander is there a way to automate license checks for dependencies in pom.xml?

Reply:
There's a Maven plugin we could use. Let me find out how to configure it.